compile/compile.go

```diff
@@ -464,6 +464,7 @@ func (c *Compiler) compilePlan(ctx context.Context) error {
 	// Create query sets for each of the entrypoints.
 	resultSym := ast.NewTerm(resultVar)
 	queries := make([]planner.QuerySet, len(c.entrypointrefs))
+	unmappedEntrypoints := ast.NewSet()
```

Comment:
[nit] If we're only storing strings, `map[string]struct{}{}` would do, too. But this is OK.

Reply:
I'll try that 👍

Reply:
Settled on `[]string` here so that the order is guaranteed.